Assignment/solution_student/aboutme.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>About Me</title>
+	<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+
+	<nav> 
+		<a href="aboutme.html">About Me</a> |
+		<a href="portfolio.html">Portfolio</a>
+	</nav>
+
+	<h1><center>JOE JOHNSON</center></h1>
+
+<hr>
+
+	<img src="images/avatar.png" alt="avatar image" align="right" class= "avatar image-border">
+
+	<h4 class= "letter spacing">I ENJOY LIFE AS A DEVELOPER</h4>
+
+	<p>I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.</p>
+<br>
+<hr>
+<br>
+<br>
```

Comment:
Same thing here with the `<br>` tags — because you're only using them for styling here, to add space, do this with CSS instead, using `padding` or `margin`